app/src/main/java/com/example/taskyy/data/local/room_entity/agenda_entities/ReminderEntity.kt

```diff
@@ -7,9 +7,9 @@ import androidx.room.PrimaryKey
 data class ReminderEntity(
     @PrimaryKey
     val id: String,
-    val userId: String?,
     val description: String,
     val remindAt: Long,
     val time: Long,
-    val title: String
+    val title: String,
+    val color: String
```

Comment:
Saving the color is dangerous. You know which agenda item type has which color, so why the need to save it? (and it's not like each item has it's own custom color which is unpredictable) 

Imagine, you want to change the colors in a future update of the app, then you'll quickly mix them with older colors if you don't have a proper DB migration